doc/en/reference.rst

```diff
@@ -976,6 +976,7 @@ passed multiple times. The expected format is ``name=value``. For example::
 
     * ``skip`` skips tests with an empty parameterset (default)
     * ``xfail`` marks tests with an empty parameterset as xfail(run=False)
+    * ``fail_at_parametrize`` raises an exception if parametrize collects an empty parameter set
```

Comment:
Also how calling the option `fail`? `fail_at_parametrize` seems redundant.

Reply:
i believe the redundancy is needed because its not a mark, it triggers a different behaviour, i believe its important to encode that in the option value

Reply:
I agree with @RonnyPfannschmidt, `fail` may be interpreted as a marker by a careless reader. A possible alternative would be to have a `fail_at_empty` option in `parametrize` itself. However, I haven't looked at that part of the code, so this is a suggestion purely based on (my) ux. This would make sense (for me), only with a default value of `True`.

Reply:
Yeah, I agree, in retrospect seems like a mistake we called the option `empty_parameter_set_mark`, `empty_parameter_set_behavior` or something like that would naturally map to `skip`, `xfail` and `failure`. Calling a mark feels to me like we leaked an implementation detail.